app/views/layouts/_nav_items.html.haml

```diff
@@ -1,3 +1,5 @@
 %ul.nav.navbar-nav
   - if current_leader
+    %li.visible-lg
+      = image_tag current_leader.user.gravatar_url, class: 'profile-image'
```

Comment:
What if they don't have a Gravatar? Would it make sense to remove this extra `%li`?